src/utils/configuration.ts

```diff
@@ -47,31 +49,33 @@ const getServerConfigurationValue = <T>(configName: keyof typeof configurationSc
     return defaultValue
 }
 
-const getRunmeTerminalConfiguration = <T>(configName: keyof typeof configurationSchema.terminal, defaultValue: T) => {
+const getRunmeTerminalConfiguration = <T>(
```

Comment:
```suggestion
const getRunmeTerminalConfigurationValue = <T>(
```

Nit, consistency with other methods